assets/js/modules/analytics/components/common/AccountCreate/index.js

```diff
@@ -49,26 +51,39 @@ import AccountField from './AccountField';
 import PropertyField from './PropertyField';
 import ProfileField from './ProfileField';
 import CountrySelect from './CountrySelect';
+import WebDataStreamField from './WebDataStreamField';
 import useViewContext from '../../../../../hooks/useViewContext';
+import { useFeature } from '../../../../../hooks/useFeature';
 const { useDispatch, useSelect } = Data;
 
 export default function AccountCreate() {
+	const ga4ReportingEnabled = useFeature( 'ga4Reporting' );
+
 	const [ isNavigating, setIsNavigating ] = useState( false );
 	const accounts = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getAccounts()
 	);
 	const hasResolvedAccounts = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).hasFinishedResolution( 'getAccounts' )
 	);
-	const accountTicketTermsOfServiceURL = useSelect( ( select ) =>
+	const accountTicketTermsOfServiceURLUA = useSelect( ( select ) =>
```

Comment:
That's quite a tough variable name to read, maybe we could use a prefix like 
```suggestion
	const uaAccountTicketTermsOfServiceURL = useSelect( ( select ) =>
```

(just because of the five capital letters in a row).

Reply:
(Same for GA4)